_po/ja/how-to/laravel.po

```diff
@@ -608,28 +923,46 @@ msgid ""
 "                                          </dd>\n"
 "                                        </div>"
 msgstr ""
+"                                        <div class=\"border-t border-gray-200 p"
+"t-1 sm:col-span-4\">\n"
+"                                          <dd class=\"sm:mt-2 text-sm text-gray"
+"-500\">\n"
+"                                              <span class=\"block m-1 line-brea"
+"k\">{{ $post->body }}</span>\n"
+"                                          </dd>\n"
+"                                        </div>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.